domain/service/musics.service.go

```diff
@@ -0,0 +1,28 @@
+package service
+
+import (
+	"github.com/Nexters/myply/domain/entity"
+	"github.com/Nexters/myply/domain/repository"
+)
+
```

Comment:
https://github.com/Nexters/myply/pull/40/files#r928938915
This is a continuation of the comment above.

I think you need to make provider(wire.NewSet(...)) in this music.service.go like configs.gi

+ nit: How about music.go? or musics.go? Because we already has service directory!

Reply:
Why don't you use the package?

The package structure of this project is a structure in which the module cannot be detached.

The package structure of this project is a structure in which the module cannot be detached.
Separate domains and place entity, repository, and service at the bottom as shown below.

![image](https://user-images.githubusercontent.com/15080806/180925277-49a9a9af-7d71-4d36-bef9-b5e7b47597aa.png)

Reply:
I saw that a lot of webserver codes made with golang put the domain layer in a folder called `pkg`, and I also thought about it.

I think that `pkg` is used considering that it will be reused somewhere later. However, in the case of a web server, since it is not reused anywhere, the directory called pkg felt unnecessary. What do you guys think? @jiwonkim-park , @devjoeween

Reply:
The word `reuse` is a magic word. And it's ambiguous.
Designing the package with `easy testing` in mind rather than `reuse` resulted in good results.